### PR TITLE
Added notes for release 4.8.29

### PR DESCRIPTION
**Pull request by ctauchen**
Notes for release 4.8.29.

Applies only to `enterprise-4.8`

https://deploy-preview-41214--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes.html#ocp-4-8-29

### CODE REVIEW

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -3009,3 +3009,28 @@ link:https://access.redhat.com/solutions/6656881[{product-title} 4.8.28 containe
...
+{product-title} release 4.8.29 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:0278[RHBA-2022:0278] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:0277[RHBA-2022:0277] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6681391[{product-title} 4.8.29 container image list]
+
+[id="ocp-4-8-29-bug-fixes"]
+==== Bug fixes
+
+* An updated Jenkins plugin, OpenShift Sync 1.0.48, wrongly specified which ConfigMap and ImageStream labels would apply to pod templates for the Kubernetes plugin for Jenkins. As a result, OpenShift Sync stopped importing pod templates from ConfigMaps and ImageStreams that had the 'jenkins-agent' label. This update corrects the label specification, and pod templates can be imported as expected.
```
Comment (opayne1) [inline release_notes/ocp-4-8-release-notes.adoc:3027]: s/can be/are

**Review by opayne1 — COMMENTED** [review]
One small nit, but otherwise lgtm. I'll have another peer reviewer come in behind me to check it since this is my first official week.

File: release_notes/ocp-4-8-release-notes.adoc (hunk 1 above)
Comment (bmcelvee) [inline release_notes/ocp-4-8-release-notes.adoc:3027]: s/ConfigMap/config map
s./ImageStream/image stream

For context, we used to write them as ConfigMap and ImageStream, but our guidelines changed a few released ago: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#config-map and https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#image-stream

Comment (ctauchen) [reply release_notes/ocp-4-8-release-notes.adoc:3027]: Fixed! Thanks!

Comment (ctauchen) [reply release_notes/ocp-4-8-release-notes.adoc:3027]: Yes, thanks, and I prefer the new guidelines. Changed!

**Pull request merged**